src/Native/Bootstrap/main.cpp

```diff
@@ -238,15 +238,17 @@ extern "C" void RhpUniversalTransition_DebugStepTailCall()
 {
     throw "RhpUniversalTransition_DebugStepTailCall";
 }
-extern "C" void CCWAddRef()
-{
-    throw "CCWAddRef";
-}
 
 void* RtRHeaderWrapper();
 
 #endif // CPPCODEGEN
 
+// TODO: Port Interop.Native to open
```

Comment:
I think we will want to rather:
- Move the last few remaining parts of S.P.Interop required for netstandard and plain PInvoke to S.P.CoreLib (this will make the netcoreapp public surface of S.P.CoreLib more similar between CoreCLR and CoreRT - some of the framework hacks will go away)
- Delete S.P.Interop from CoreRT, and move it into a new COM/WinRT interop support repo together with MCG

Reply:
Updated comment.